packages/typography/src/Link/Link/Link.spec.tsx

```diff
@@ -6,11 +6,7 @@ import { InferredPolymorphicProps } from '@leafygreen-ui/polymorphic';
 import Link from './Link';
 import { BaseLinkProps } from './Link.types';
 
-type SpanLikeProps = InferredPolymorphicProps<'span', BaseLinkProps>;
-
-type AnchorLikeProps = InferredPolymorphicProps<'a', BaseLinkProps>;
-
-type LinkRenderProps = SpanLikeProps | AnchorLikeProps;
+type LinkRenderProps = InferredPolymorphicProps<'span', BaseLinkProps>;
```

Comment:
Can we get away with this, because if `href` is defined, the component props are set to `a` element props, and `span` is ignored?